### DISCUSSION TITLE
How to load js from remote storage ?

### DISCUSSION THREAD

**Discussion by echobinod** [GrapesJS/grapesjs#3847] [Q&A]
Hello @artf,
Response to load html, css is like this.
How to load JavaScript for specific page and whole pages. What type of response should be returned to load JS as well ?
```
{
    "pages": [
        {
            "id": "page2",
            "name": "page1.html",
            "component": "<div w3-include-html=\"page2.html\"></div>\n\n<h1>Page 1</h1>",
            "css": ""
        },
    ],
    "styles": "body{background-color: red;}"
}
```

**Comment by hela-Ch** [comment]
@echobinod  please how did you load multiple pages from remote? which type of response returned to load and to store? Thank you :)
